.github/workflows/test_frameworks.yml

```diff
@@ -94,6 +94,39 @@ jobs:
         # django.tests.requests module interferes with requests library patching in the tracer -> disable requests patch
         run: DD_TRACE_REQUESTS_ENABLED=0 ddtrace-run tests/runtests.py
 
+  graphne-testsuite-3_0:
+    runs-on: ubuntu-latest
+    env:
+      DD_PROFILING_ENABLED: true
+      DD_TESTING_RAISE: true
+    defaults:
+      run:
+        working-directory: graphene
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          path: ddtrace
+      - uses: actions/checkout@v3
+        with:
+          repository: graphql-python/graphene
+          ref: v3.0.0
+          path: graphene
+      - uses: actions/setup-python@v3
+        with:
+          python-version: "3.10"
+      - name: Install ddtrace
+        run: pip install ../ddtrace
+      - name: Install graphene
+        run: pip install -e "../graphene[test]"
+      - name: "Upgrade pytest"
+        # pytest<6.2.5 raises TypeError: required field "lineno" missing from alias on python 3.10
+        run: pip install "pytest>=6.2.5,<7"
+      - name: Set Pythonpath
+        run: echo "PYTHONPATH=." >> $GITHUB_ENV
+      - name: Run tests
+        # skip broken tests
```

Comment:
can we add a brief line for why each test is disabled so that we could possibly reenable the test later on?